docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>"The numba function operator can take up to 6 inputs and produce up to 6 outputs."</p><p> </p><p>"As an example, let us write a pipeline that takes an image and splits the RGB channels into 3 separate outputs."</p><p> </p><p>"First, we need to define the setup function that defines the shapes of the outputs."</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>